pkg/kyma/docs/install.md

```diff
@@ -14,17 +14,17 @@ kyma install [OPTIONS]
 
 | Name     | Short Name | Default value| Description|
 | ----------|---------|-----|------|
-| --release | -r ||Kyma release to use|
-| --local | -r |false|Install from sources| 
-| --config | -r ||URL or path to the Installer configuration YAML file| 
-| --override | -r ||Path to YAML file with parameters to override. Multiple entries of this flag are allowed| 
-| --domain | -r |kyma.local|Domain used for installation| 
-| --password | -r ||Predefined cluster password| 
-| --noWait | -r |false|Do not wait for the Installer configuration to complete| 
-| --src-path | ||Path to local sources| 
-| --installer-version | ||Version of the Kyma Installer Docker image used for local installation| 
-| --installer-dir | ||The directory of the Kyma Installer Docker image used for local installation| 
-| --timeout |  |0|Timeout after which CLI stops watching the installation progress| 
+| --release | -r ||Specifies the Kyma release or git revision to be installed. Go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about each of the available releases, or use the revision of your choice. For example, `kyma install --release master`.|
+| --local | -r |false|Indicates installtion from sources. To make it work, make sure you have **{GO_PATH}** set and the location of your sources complies with the Go file and naming convention.| 
+| --config | -r ||Specifies the URL or path to the Installer configuration YAML file.| 
+| --override | -r ||Specifies the path to YAML file with parameters to override. Multiple entries of this flag are allowed.| 
+| --domain | -r |kyma.local|Specifies the domain used for installation.| 
+| --password | -r ||Specifies the predefined cluster password.| 
+| --noWait | -r |false|Determines if the installation should wait for the Installer configuration to complete.| 
+| --src-path | ||Specifies the absolute path to local sources.| 
+| --installer-version | ||Specifies the version of the Kyma Installer Docker image used for the local installation.| 
+| --installer-dir | ||Specifies the directory of the Kyma Installer Docker image used for the local installation.| 
+| --timeout |  |0|Specifies the timeout after which CLI stops watching the installation progress.| 
```

Comment:
default gets currently changed to 30m, please check with @clebs the notation for values, it must be mentioned here